src/components/Tooltip/BaseTooltip.tsx

```diff
@@ -74,11 +87,11 @@ function Tooltip({children, numberOfLines, maxWidth, text, renderTooltipContent,
     const isAnimationCanceled = useRef(false);
     const prevText = usePrevious(text);
 
-    const target = useRef(null);
+    const target = useRef<HTMLElement | null>(null);
     const initialMousePosition = useRef({x: 0, y: 0});
 
-    const updateTargetAndMousePosition = useCallback((e) => {
-        target.current = e.currentTarget;
+    const updateTargetAndMousePosition = useCallback((e: MouseEvent) => {
+        target.current = e.currentTarget as HTMLElement;
         initialMousePosition.current = {x: e.clientX, y: e.clientY};
```

Comment:
Can we do this? You should test it first though
```suggestion
    const updateTargetAndMousePosition = useCallback((e: MouseEvent) => {
        if (e.currentTarget instanceof HTMLElement) {
            target.current = e.currentTarget;
            initialMousePosition.current = {x: e.clientX, y: e.clientY};
        }
```